sw/airborne/firmwares/rotorcraft/navigation.h

```diff
@@ -111,6 +111,8 @@ extern bool_t nav_set_heading_current(void);
 #define NavSetWaypointHere(_wp) ({ waypoint_set_here_2d(_wp); FALSE; })
 #define NavCopyWaypoint(_wp1, _wp2) ({ waypoint_copy(_wp1, _wp2); FALSE; })
 #define NavCopyWaypointPositionOnly(_wp1, _wp2) ({ waypoint_position_copy(_wp1, _wp2); FALSE; })
+#define NavSetWaypointPos(_wp, _enu_x, _enu_y) ({ waypoint_set_xy(_wp, _enu_x, _enu_y); FALSE; })
```

Comment:
What's the reason for adding these?
I'm not in favour of adding more macros unless needed...